ruma-events/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 Breaking changes:
 
+* Change `get_message_events` limit field type from `Option<Value>` to `Map<String, Value>`
```

Comment:
This seems to be a copy-pasted entry that you forgot to fully rewrite?

Reply:
Ah sorry and thanks for catching this.